regexes.go

```diff
@@ -30,6 +30,7 @@ const (
 	latitudeRegexString            = "^[-+]?([1-8]?\\d(\\.\\d+)?|90(\\.0+)?)$"
 	longitudeRegexString           = "^[-+]?(180(\\.0+)?|((1[0-7]\\d)|([1-9]?\\d))(\\.\\d+)?)$"
 	sSNRegexString                 = `^\d{3}[- ]?\d{2}[- ]?\d{4}$`
+	hostnameRegexString            = `^(([a-zA-Z]|[a-zA-Z][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z]|[A-Za-z][A-Za-z0-9\-]*[A-Za-z0-9])$`
```

Comment:
Also minor but if you could make the groups non-capturing that'd be great! eg. (?: